source/includes/administration/_commitments.md

```diff
@@ -467,4 +467,71 @@ Attributes | &nbsp;
     "taskId": "75a68ec2-d63e-4bc9-b485-8d5a3cc33d4f",
     "taskStatus": "SUCCESS"
 }
-```
\ No newline at end of file
+```
+
+<!-------------------- UPDATE COMMITMENT -------------------->
+### Update commitment
+
+`PUT /commitments/:id`
+
+Update the attributes of a commitment for an organization
+
+A commitment that is applied can be updated. Meaning, that since an invoice hasn't been issued to the customer, all of the attributes of the commitment can be updated.
+
+A commitment that is pre-paid can be updated, **but** it will be terminated and a new commitment will be created with the updated attributes and will be effective at the start at the end of the current billing cycle.
+
+Updating a commitment always requires a reprocess from the start of the current billing cycle, expect if only the name has been updated.
```

Comment:
I don't think mentioning the reprocess in the API docs is worthwhile. It doesn't impact how the user interacts with the API.

Reply:
Done 130920581335d8572ed8f1c5ff2ebb5dd6004ad0